src/main/scala/mesosphere/marathon/api/akkahttp/Directives.scala

```diff
@@ -98,6 +101,31 @@ object Directives extends AuthDirectives with LeaderDirectives with AkkaDirectiv
     }
   }
 
+  def extractTaskKillingMode: Directive1[TaskKillingMode] =
```

Comment:
put it into `AppDirectives` may be? Let's try not to create a `UtilsDirective` class - those always end up being a big pile of "garbage"